Concept_Templates/crystalline_material.md

```diff
@@ -0,0 +1,42 @@
+## Knowledge domain resources
+
+### Related
+
+https://dictionary.iucr.org/crystal: historical, reciprocal and direct space definitions. Expert discussions
+also raised the issue of different dimensionality crystals, quasicrystals and so on.
+
+https://en.wikipedia.org/wiki/Crystal: information as above, as well as history of the term,
+some discussion of various types of crystal including quasicrystals and crystal defects.
+
+### Comments
+
+The vast majority of crystal structures are formed of arrangements of atoms that are periodic in
+3 dimensions (400,000 according to Wikipedia reference 20).
+
+## General Concept Info
+
+### IRI: TBA
+### OWL type: TBA
+### Concept elucidation
+A solid substance for which measurements on a sample of that substance have 
+been well-described by using the crystalline model (see #5 ). The model used
+may include a number of distinct crystal models which taken together completely
+describe the measurements. This concept does not include materials for which a
+non-crystalline model has been included (for example, an amorphous component)
+in order to describe the measurements.  This concept does include materials
+for which the models used include other parameters not related to the atomic
+structure of the material, for example powder diffraction modelling often
+involves modelling of the instrumental contribution to the measurements.
+### Preferred label
+"crystalline material"
+### Alternate labels
+ "crystal", "three-dimensional crystal", "periodic crystal", "powder"
```

Comment:
I am little concerned about "powder" as alternative label, both because we discuss powder as an own concept but also because a powder does not need to be crystalline in my vocabulary (I would e.g. call fine crushed glass for a powder). The latter depends on what we mean with powder.

Reply:
How about "crystalline powder" instead?